### PR TITLE
I172 python3 rewrite

### PR DESCRIPTION
**Pull request by AlexejPopovic**

### CODE REVIEW

**Comment by AlexejPopovic** [comment]
@cclauss Thank you. Fixed in 6012e8c0c5ad39c429633e183c77ba0aa2b9f360 and 86bb5b0562f8eb08a9d0533e671a4bdd998838b5.

File: scripts/cfgCommons.py [hunk 1]
```diff
@@ -49,7 +49,9 @@ def __init__(self, args):
                 for common_configFilePath in args.common_configFilePaths: # go over all the config files and collect all parameters
                     logger.info("Processing config file:" + common_configFilePath)
                     configPart = configparser.ConfigParser()
-                    configPart.read(common_configFilePath)
+                    with openFile(common_configFilePath) as configFile:
+                        configPart.read_file(openFile(common_configFilePath))
```
Comment (ondrejtomas) [inline scripts/cfgCommons.py]: I guess that we want to change it to something like: `configPart.read_file(configFile)`

Comment (AlexejPopovic) [reply scripts/cfgCommons.py]: Right, fixed in a454042 .

File: scripts/workspace_deploy.py [hunk 2]
```diff
@@ -52,8 +52,8 @@ def main(argv):
 
     # workspace info
     try:
-        workspaceFilePath = os.path.join(getattr(config, 'common_outputs_directory'), getattr(config, 'common_outputs_workspace'))
-        with openFile(workspaceFilePath, 'r') as workspaceFile:
+        workspaceFilePath = os.path.join(getRequiredParameter(config, 'common_outputs_directory'), getRequiredParameter(config, 'common_outputs_workspace'))
+        with openFile(workspaceFilePath, 'r') as workspaceFile
```
Comment (cclauss) [inline scripts/workspace_deploy.py]: The colon (:) is essential.

Comment (ondrejtomas) [reply scripts/workspace_deploy.py]: Thank you, but I have already fix that 4 mins before you. :)

**Pull request merged**